app/services/auth.js

```diff
@@ -83,6 +83,29 @@ export default Service.extend({
 
   redirectUrl: null,
 
+  init() {
+    this._super(...arguments);
+    window.addEventListener('focus', this.checkAuth.bind(this));
+  },
+
+  checkAuth() {
+    if (!this.currentUser || !this.storage)
+      return;
+    const { accounts } = this.storage;
+    const { vcsId } = this.currentUser;
+    let stillLoggedIn = false;
+
+    accounts.map((user) => {
+      if (user.vcsId === vcsId) {
+        stillLoggedIn = true;
+      }
+    });
```

Comment:
There's a simpler way to do this with Ember:
```suggestion
    const stillLoggedIn = accounts.isAny('vcsId', vcsId);
```
Here's more info: https://api.emberjs.com/ember/3.12/classes/EmberArray/methods/isAny?anchor=isAny

Reply:
Awesome, thank you @artursmirnov !